### DISCUSSION TITLE
Where are new icons?

### DISCUSSION THREAD

**Discussion by vokic** [primefaces/community#996] [PrimeIcons] [answered]
Hi, I don't want to sound rude, but has any of the icons been added per [requests](https://github.com/primefaces/primeicons/issues)? Am I looking at a different set in primeNg? Is there any collection that is more up to date than this?

https://primeng.org/icons

Also, I want to contribute in creating icons if that is an issue.

✌

**Comment by cetincakiroglu** [comment]
@onursenture

**Comment by onursenture** [comment, marked as answer]
The icons in the linked URL are PrimeIcons 6.0.1, which means they are all icons in the latest version. We are aware that numerous icon requests have accumulated in the issue tracker.

We update PrimeIcons at each year's end, but changes in our workflow have pushed this year's update to the first quarter of 2024. We have previously used Adobe Illustrator to make icons; however, we are now shifting our entire icon creation process to Figma. We apologize for the delay. This switch will enable us to deliver updates more swiftly than before.

**Reply by vokic** [reply]
If you need any help regarding design, feel free to contact me 🙂
